packages/ui/react-ui-theme/src/styles/layers/attention.css

```diff
@@ -1,24 +1,29 @@
 @layer utilities {
+  /* If an element is attendable, it should have this class. */
   .attention-surface {
-    --surface-bg: var(--dx-attention);
+    --surface-separator: transparent;
     background-color: var(--surface-bg);
+    border-color: var(--surface-separator);
   }
 }
 
+/* Utilities below control _how_ an element becomes attended. */
 @layer components {
-  .attention-within:focus-within {
+  /* If an element is attended this attribute will be set. */
+  [data-is-attention-source] {
     --surface-bg: var(--dx-attention);
     --surface-separator: var(--dx-separator);
   }
 
-  .group\/attention:not(:focus-within) .attention-current[aria-current] {
-    --surface-bg: var(--dx-attention);
-    --surface-separator: var(--dx-separator);
-  }
-
-  .attention-static,
-  .attention-within.attention-static {
-    --surface-bg: var(--dx-attention);
-    --surface-separator: var(--dx-separator);
+  /* Used to style elements which are _related_ to the currently attended element. */
+  .current-related[aria-current] {
+    /*
+      NOTE(wittjosiah): Current these colors are the same, but they could be different.
+        If they are different, a background transition is likely required to avoid flashing.
+    */
+    --surface-bg: var(--dx-currentRelated);
+    &:focus-within {
+      --surface-bg: var(--dx-attention);
```

Comment:
Ah interesting… I was anticipating this should be a different selector or rule, but this actually seems more cogent, and if you’ve tested it and it works then this seems ideal.